.github/workflows/main.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         exclude:
```

Comment:
exclude test a `dc: ldc-1.25.0, os: windows-2019`

Reply:
Sorry? I don't follow.

Reply:
os: [ ubuntu-18.04, macOS-10.15 ]  just run :)

Reply:
Ahh you mean to not use exclude. Right.

Reply:
Thank you @TrustHenry the Wise.